egs2/minds14/asr1/README.md

```diff
@@ -0,0 +1,62 @@
+---
+tags:
+- espnet
+- audio
+- automatic-speech-recognition
+language: es-ES
+license: mit
+---
+# Dataset
+* MInDS-14: https://aclanthology.org/2021.emnlp-main.591/
+
+### Supported Languages
+* format: `{lang}-{LOCALE}`
+* languages
+   * `cs-CZ`
+   * `de-DE`
+   * `en-AU`
+   * `en-GB`
+   * `en-US`
+   * `es-ES`
+   * `fr-FR`
+   * `it-IT`
+   * `ko-KR`
+   * `nl-NL`
+   * `pl-PL`
+   * `pt-PT`
+   * `ru-RU`
+   * `zh-CN`
+---
+
+# How to run
+* In `run.sh`, edit the value of `language` to `{lang}-{LOCALE}` (ex. `es-ES`, `en-AU`)
+---
+
+# RESULTS

```

Comment:
Do you have more results (SSLR etc.) and corresponding pretrained models?

Reply:
![Screen Shot 2022-04-04 at 10 20 14 PM](https://user-images.githubusercontent.com/35610230/161666371-143560b8-8018-42b3-a3be-2148db5a629b.png)

This has all the runs and results . Due to compute limitations we could not tune to get good performance with SSL.